app/screens/AppNavigator/BottomTabNavigator.tsx

```diff
@@ -31,12 +28,9 @@ function TabBarIcon (props: { name: VectorIconName, color: string }): JSX.Elemen
 }
 
 export function BottomTabNavigator (): JSX.Element {
-  const isOceanInterfaceOpen = useSelector((state: RootState) => isDrawerOpenSelector(state.ocean))
   return (
     <>
-      {
-        isOceanInterfaceOpen && <OceanInterface />
-      }
+      <OceanInterface />
```

Comment:
The issue was component attempt to render when the queue is empty. Should be okay now.

Reply:
This might be the cause also why the animation doesn't work anymore. I'll check on how we can move it on the new structure.